test_mcp_ecosystem_expansion.py

```
@@ -55,434 +58,475 @@ async def run_all_tests(self) -> Dict[str, Any]:
+                    "error": str(e),
+                    "timestamp": datetime.utcnow().isoformat(),
                 }
                 self.test_results[suite_name] = error_result
                 logger.error(f"❌ {suite_name}: {str(e)}")
-        
+
         # Generate final report
         final_report = self._generate_final_report()
-        
+
         logger.info("\n" + "=" * 60)
```

Ruff (0.11.9)

Logging statement uses `+`

(G003)